src/parser/xasl_generation.c

```diff
@@ -4205,6 +4205,19 @@ pt_to_aggregate_node (PARSER_CONTEXT * parser, PT_NODE * tree, void *arg, int *c
 		  aggregate_list->flag_agg_optimize = true;
 		}
 	    }
+	  else
```

Comment:
Does PT_MIN, PT_MAX only enter the else clause? I'm asking you in connection with the PT_COUNT that I already asked you above.

Reply:
As I mented as above, PT_COUNT optimize will be handled in the future.